hack/run_in_cloud.sh

```diff
@@ -0,0 +1,75 @@
+#!/bin/bash
+# Note: Tested only with buildah / podman
```

Comment:
we don't need to support docker for our internal stuff. If you have only tested this with podman, just make it only work with podman

Reply:
done